### PR TITLE
S83: line with two origins

### PR DESCRIPTION
**Pull request by prabau**
New space: line with two origins.

### CODE REVIEW

File: properties/P000024.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ refs:
     name: Locally compact space on Wikipedia
 ---
 
-Each point has a basis of neighborhoods relatively compact to the space.
+Each point has a local base of neighborhoods relatively compact to the space (that is, neighborhoods with compact closure).
```
Comment (StevenClontz) [inline properties/P000024.md:13]: I generally call such subsets "precompact" and reserve "relatively compact" for "every open cover of the *space* has a finite subcover for the *subset*". These are equivalent assuming regularity, but not in general.

So if the intent here is actually "a local base of neighborhoods with compact closures", then I think we should just say that and avoid the ambiguity.

**Review by StevenClontz — APPROVED** [review]

File: properties/P000024.md (hunk 1 above)
Comment (StevenClontz) [reply properties/P000024.md]: I made this change and approved, so go ahead and merge if you're happy with the tweak.

Comment (StevenClontz) [reply properties/P000024.md]: Hmm I guess the name "locally relatively compact" is unclear as well, but maybe that can be handled separate from the PR.

**Comment by prabau** [comment]
Your change looks good.  The reason I wanted to add this clarification (or the change you made) is like you said, the possible ambiguity between "precompact" and "relatively compact", especially since we used "relatively compact" with a different meaning than here for property https://topology.pi-base.org/properties/P000071.

As for changing the name of the property itself, that could possibly be done in a separate PR.  But it's not entirely clear what the best is here.  "Relatively compact" and "precompact" are often used interchangeably, usually with the meaning of "has compact closure".  "Locally relatively compact" has been picked up in Wikipedia (like you mention in your changes for #417), and used occasionally in the literature (see refs in Wikipedia for example).  On the other hand, a google search seems to show that "locally precompact" is used in the context of topological groups with a completely different meaning?
It's kind of confusing.

In the mean time, I'll merge this.

**Pull request merged**